cypress/integration/common/elements.js

```diff
@@ -12,17 +12,39 @@ const MapNumberTextToNumber = {
 // map from natural language text to HTML tag name
 const MapElementSpecifierToTag = {
     'level 1 heading': 'H1',
-    'level 2 heading': 'H2'
+    'level 2 heading': 'H2',
```

Comment:
not sure if this was intentional - but there's an additional comma here. i know PHP allows this...